stdlib/2and3/genericpath.pyi

```diff
@@ -1,14 +1,17 @@
-from typing import List
+from typing import List, AnyStr, Text
 import sys
 
-def exists(path: str) -> bool: ...
-def isfile(path: str) -> bool: ...
-def isdir(s: str) -> bool: ...
-def getsize(filename: str) -> int: ...
-def getmtime(filename: str) -> float: ...
-def getatime(filename: str) -> float: ...
-def getctime(filename: str) -> float: ...
-def commonprefix(m: List[str]) -> str: ...
+if sys.version_info < (3, 0):
+    class _unicode: ...
+
+def exists(path: Text) -> bool: ...
+def isfile(path: Text) -> bool: ...
+def isdir(s: Text) -> bool: ...
+def getsize(filename: Text) -> int: ...
+def getmtime(filename: Text) -> float: ...
+def getatime(filename: Text) -> float: ...
+def getctime(filename: Text) -> float: ...
+def commonprefix(m: List[AnyStr]) -> AnyStr: ...
```

Comment:
commonprefix() should not be generic in Python 3, it only works for strings. (Witness the `return ''`) in the source code.